imagenet/main.py

```diff
@@ -215,6 +215,9 @@ def main_worker(gpu, ngpus_per_node, args):
                 best_acc1 = best_acc1.to(args.gpu)
             model.load_state_dict(checkpoint['state_dict'])
             optimizer.load_state_dict(checkpoint['optimizer'])
+            if args.lr:
```

Comment:
wouldn't `if args.lr` always evaluate to true?

Reply:
shoot, yes, thank you for pointing that out! I must have been looking at `start_epoch` or something